app/javascript/utilities/generate-utility-classes-documentation.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+const {
+  GENERATED_STORIES_FOLDER,
+  getStyleSheet,
+  generateUtilityClassesDocumentation,
+} = require('./documentation');
+
+const folderExists = util.promisify(fs.exists);
+const mkdir = util.promisify(fs.mkdir);
+const stylesheetsDirectory = path.resolve(
+  __dirname,
+  '../../assets/stylesheets',
+);
+
+(async () => {
```

Comment:
The current version of node.js that we are using does not support top level await, hence this async [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE).